README.md

```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
 
 ## Submit a model
 
+1. Create a GitHub account if you do not have one yet : [https://github.com/join](https://github.com/join).
+
+    Learn more about GitHub, please visit [Introduction to Git](https://docs.microsoft.com/en-us/learn/modules/intro-to-git/)
+
+1. Please sign [Contributor License Agreement](https://cla.opensource.microsoft.com/Azure/iot-plugandplay-models?pullRequest=93)
+
 1. Fork the public GitHub repo: [https://github.com/Azure/iot-plugandplay-models](https://github.com/Azure/iot-plugandplay-models).
+
+    Learn more about forking a repo : [https://docs.github.com/en/github/getting-started-with-github/fork-a-repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)
+
 1. Clone the forked repo. Optionally create a new branch to keep your changes isolated from the `main` branch.
-1. Add the new interfaces to the `dtmi` folder using the folder/filename convention. See the [add-models](#import-a-model-to-the-dtmi-folder) tool below.
+
+    By forking and cloning the public GitHub repo, a copy of repo will be created in your GitHub account and a local copy is created in your dev machine.  Please use this local copy to make modifications.
+
+1. Author a new device model with an unique ID using [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier).  
+
+    Review the [PR requirements](pr-reqs.md) for naming conventions.
+
+1. Save the device model JSON file to a local folder.  
```

Comment:
suggestion to link to the DTDL VSCode extension, to help users creating new interfaces

Reply:
Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)